frontend/src/js/App.js

```diff
@@ -43,9 +43,23 @@ let store = configureStore();
         $(this).on("DOMNodeInserted", child_inserted);
         return this;
 
-
     };
+}(jQuery));
 
+(function(jQuery) {
```

Comment:
This provides a regex selector for jquery so I can do a regex search for highlighting row or columns when moving headers.